shell/common/common_derby.sh

```diff
@@ -1,3 +1,5 @@
+source_file "$ALOJA_REPO_PATH/shell/common/common_java.sh"
+set_java_requires
```

Comment:
Aren't these just the lines that you removed in the first commit? If so, just modify that commit and repush the branch - we don't need to follow the story of "remove, then bugfix (add them back)". You can just rewrite the history and pretend that it never happened.